[*] Running ...
[*] Dataset loaded: E:\Coding\pythonProject\llm4contract\problems\dataset\train_01.pkl with 5 instances.
â†’ Best principal utility: 4.03773573646638
Traceback (most recent call last):
  File "E:\Coding\pythonProject\llm4contract\problems\eval.py", line 158, in <module>
    obj = evaluate_llm((_p, _c, _v, _contract_logs))
  File "E:\Coding\pythonProject\llm4contract\problems\eval.py", line 102, in evaluate_llm
    inferred_agent_setting = llm_agent_solver(v, content)
  File "E:\Coding\pythonProject\llm4contract\problems\gpt.py", line 76, in agent_solver
    for k in range(n_clusters):
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 5 is different from 97)
